.github/workflows/test.yml

```diff
@@ -464,15 +464,15 @@ jobs:
           - {"target": "network-segmentation", "ha": "noHA", "gateway-mode": "shared", "ipfamily": "ipv4", "disable-snat-multiple-gws": "noSnatGW", "second-bridge": "1br", "ic": "ic-single-node-zones"}
           - {"target": "network-segmentation", "ha": "noHA", "gateway-mode": "shared", "ipfamily": "ipv6", "disable-snat-multiple-gws": "noSnatGW", "second-bridge": "1br", "ic": "ic-single-node-zones"}
           - {"target": "bgp", "ha": "noHA", "gateway-mode": "local",  "ipfamily": "dualstack", "disable-snat-multiple-gws": "snatGW", "second-bridge": "1br", "ic": "ic-single-node-zones", "routeadvertisements": "advertise-default", "network-segmentation": "enable-network-segmentation"}
-          - {"target": "bgp", "ha": "noHA", "gateway-mode": "shared",  "ipfamily": "ipv4", "disable-snat-multiple-gws": "snatGW", "second-bridge": "1br", "ic": "ic-single-node-zones", "routeadvertisements": "advertise-default", "network-segmentation": "enable-network-segmentation"}
```

Comment:
```
Make both dualstack, not much value testing IPv4 single stack
```
are you sure? sometimes singlestack v6 is forgotten to be handled by folks but perhaps singlestackv4 in itself doesn't add though it is the most used setup today from what I know.

Reply:
```
 It is useful to test for IPv4 specific bits in our code but there are not much and are already covered nicely in unit tests. So I think adding IPv6 coverage to both gateway modes adds more value.
```
ack agree I would rather have a new lane added is what i meant than removing v4 but its fine. I don't have a strong opinion